Frontend/ui-driver/src/Helpers/Utils.purs

```diff
@@ -237,11 +237,16 @@ capitalizeFirstChar inputStr =
     in DS.joinWith " " output
 
 getDowngradeOptions :: String -> Array String
-getDowngradeOptions vehicleType = case vehicleType of
-  "SEDAN" -> ["HATCHBACK"]
-  "SUV" -> ["HATCHBACK", "SEDAN"]
-  "TAXI_PLUS" -> ["TAXI"]
-  _ -> []
+getDowngradeOptions variant = case getMerchant FunctionCall of
+                                YATRISATHI -> case variant of
+                                                "TAXI" -> []
+                                                "SUV"  -> ["HATCHBACK", "SEDAN"]
```

Comment:
@ashishsingh101 as SUV is AC SUV here so according to figma the downgrading option available to them should be AC Cab which is TAXI_PLUS.

Reply:
@MercyQueen downgrading options are same as before, from backend we have only three options to downgrade on, we don't have option for `TAXI_PLUS `